elasticsearchIndexing/pyang_plugin/json_tree.py

```diff
@@ -228,11 +228,15 @@ def get_nontypedefstring(node):
             if t.arg == 'enumeration':
                 s['enumeration'] = [enums.arg for enums in t.substmts]
 
-            elif t.arg == 'leafref' and (p := t.search_one('path')) is not None:
-                s['path'] = p.arg
-
-            elif t.arg == 'identityref' and (b := t.search_one('base')) is not None:
-                s['base'] = b.arg
+            elif t.arg == 'leafref':
+                p = t.search_one('path')
+                if p is not None:
+                    s['path'] = p.arg
+
+            elif t.arg == 'identityref':
+                b = t.search_one('base')
+                if b is not None:
+                    s['base'] = b.arg
```

Comment:
this part is rewritten this way in case we need to use ```pypy3``` in the future, it doesn't support the ```:=``` operator yet, so it will fail on every run of the ```pypy3``` then even if another plugin is run